.github/workflows/test-spras.yml

```diff
@@ -155,6 +158,18 @@ jobs:
         tags: v2
         cache_froms: reedcompbio/py4cytoscape:v2
         push: false
+  #Adding LN Here
+    - name: Build LocalNeighborhood Docker image
+      uses: docker/build-push-action@v1
+      with:
+        path: docker-wrappers/LocalNeighborhood/.
+        dockerfile: docker-wrappers/LocalNeighborhood/Dockerfile
+        #Ask about the following line
+        repository: reedcompbio/localneighborhood
```

Comment:
Cache from your dwnachre/local-neighborhood and also use that for the repository name here because there isn't a reedcompbio image for this algorithm.